contracts/contracts/RequestManager.sol

```diff
@@ -247,16 +291,19 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
     /// @param _claimRequestExtension Extension to claim a request after validity period ends.
     /// @param _claimPeriod Claim period, in seconds.
     /// @param _challengePeriodExtension Challenge period extension, in seconds.
+    /// @param _lpMarginPPM Liquidity provider profit margin.
     constructor(
         uint96 _claimStake,
         uint256 _claimRequestExtension,
         uint256 _claimPeriod,
-        uint256 _challengePeriodExtension
+        uint256 _challengePeriodExtension,
+        uint256 _lpMarginPPM
```

Comment:
I'm wondering about whether we need this in the constructor, or rather, whether lpMarginPPM needs to be immutable. For instance, we have
```
        uint32 lpFeePPM;
        uint32 protocolFeePPM;
```
as part of `Token`, which I was wondering about above.
If we don't really need these two values to be per-token, or per-chain, then we can consider all three

```
        uint32 lpFeePPM;
        uint32 protocolFeePPM;
        uint32 lpMarginPPM
```
as a separate contract-level part of our fee model. (I don't have a nice name for such a struct, but we probably can also go without the struct too...) Also, this would allow `lpMarginPPM` to be mutable the same way the other two fields are.

Reply:
Good point, I'm thinking about why we decided to put it into the token struct.
At the end, it's the users demand which will decide if different lpFees are necessary for different tokens. 
For now, I'm actually happy to move them out of the struct and have it protocol wide.

Regarding mutability, I do see that the lpFee and protocolFee are more subject to be changed regularly than the lpMargin on the minLpFee, but this also may be just my subjective view.
I have nothing against making it mutable.

Reply:
> Good point, I'm thinking about why we decided to put it into the token struct, It probably just moved their because of the minLpFee which was token dependent.
> At the end, it's the users demand which will decide if different lpFees are necessary for different tokens.
> For now, I'm actually happy to move them out of the struct and have it protocol wide.

Cool, then let's have all three of these
```
        uint32 lpFeePPM;
        uint32 protocolFeePPM;
        uint32 lpMarginPPM
```
on the contract level and move `lpMarginPPM` out of the constructor.
We would then need another function to update these, e.g. `updateFees`, right?
So we end up with
```
function updateToken(address tokenAddress, uint256 transferLimit, uint256 ethInToken)
function updateChain(uint256 chainId, uint256 finalityPeriod, uint256 transferCost, uint256 targetWeightPPM)
function updateFees(uint32 lpFeePPM, uint32 protocolFeePPM, uint32 lpMarginPPM)
```
where that last one would be called the least often.

So after deployment we call all three to configure the contract, and then only the first two(?) on a, say, weekly basis.